website/community/product-steering/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Product Steering
+weight: 10
+---
+
+## Product Steering
+
+### What is Gardener's Product Steering?
+Product Steering provides strategic direction for major product increments. Product ideas can be submitted to the Product Steering Committee as proposals. The proposer must provide a brief document outlining the idea. Ideally, if applicable, a prototype should be included to demonstrate feasibility, but no full implementation or design document is expected (or desired) at this stage.
+
+The committee will schedule a Product Steering meeting, where the proposer presents the proposal. Committee members, interested contributors, and affected sponsors will challenge and discuss the idea. This meeting will be recorded.
+
+Following this, a private go/no-go decision meeting with the affected sponsors will take place. This second meeting will not be recorded, but the outcome will be documented and shared here.
+
+### Why Do We Need It?
+We aim to align sponsored development capacity with Gardener's strategic vision and priorities. Investments should deliver value either to Gardener's users and operators or to the project and its overall reach.
+
+### Who is on The Product Steering Committee?
+The committee is elected by the project's active contributing sponsors and consists of a small number of members (at least three). Currently, the committee members are:
+- [Johannes Scheerer](https://github.com/scheererj) ([Email](mailto:johannes.scheerer@sap.com) and [Slack](https://gardener-cloud.slack.com/archives/D08EXK51QJJ))
+- [Rafael Franzke](https://github.com/rfranzke) ([Email](mailto:rafael.franzke@sap.com) and [Slack](https://gardener-cloud.slack.com/archives/DAQ7R4D6D))
+- [Vedran Lerenc](https://github.com/vlerenc) ([Email](mailto:vedran.lerenc@sap.com) and [Slack](https://gardener-cloud.slack.com/archives/DAQH1NTUL))
+
+### How Does It Work?
+#### 1. Proposal Submission
+- A proposer drafts a **2-5 page product proposal** detailing the idea and shares a link to it with any member of the Product Steering Committee (see contact data above).
+- The proposal is linked on this community page, and a **meeting date** is scheduled **1-4 weeks out** (at least one week for review and preparation but no more than four weeks to ensure timely decisions).
+
+#### 2. Participation
+- Those who wish to **actively challenge or discuss** the proposal can **request an invitation** from any committee member.
```

Comment:
How will interested people get to know about an upcoming proposal discussion?
I'd say we should have a Slack channel, where upcoming meetings are announced (a single one is probably sufficient for now). In addition, we could announce this via our mailing list.